src/cozy/services/sharing.service.ts

```diff
@@ -143,6 +174,32 @@ export class SharingService {
         return organizationsWithoutCozy;
     }
 
+    protected async loadTrustedUsers() {
+        const queryDefinition = Q('com.bitwarden.contacts')
+            .select(['email', 'public_key', 'confirmed'])
+            .where({confirmed: true});
```

Comment:
Do not hesitate to add an `indexFields` sorted by the most restrictive field first https://docs.cozy.io/en/tutorials/data/queries/#index-fields

Reply:
After reading the `indexFields` documentation:
- `confirmed` is not always present in `com.bitwarden.contacts`
  - My understanding is that I can create and index on `confirmed`, the requirement is that I don't use `$exists` on it. Is that correct?
- There is no need to add `indexField` for fields that are not present in the `where` condition. Do you confirm that?

Reply:
If the field is not always present, then you can read the documentation about partial indexes https://docs.cozy.io/en/tutorials/data/advanced/#partial-indexes :D 

> There is no need to add indexField for fields that are not present in the where condition. Do you confirm that?

yep

Reply:
`.indexFields(["confirmed"])` has been added in last forced push